locale/config.js

```diff
@@ -1,159 +1,43 @@
-//! moment.js locale configuration
-//! locale : Japanese [ja]
-//! author : LI Long : https://github.com/baryon
+var e = {
```

Comment:
Please use "dayjs.locale" directly like other language pack. Additionally, please provide the citation.
他の翻訳パックと同様に直接"dayjs.locale"を使用してください。また、引用元を示してください。
```suggestion
// Based on: https://github.com/iamkun/dayjs/blob/dev/src/locale/ja.js

dayjs.locale({
```